src/fb5.js

```diff
@@ -234,18 +235,51 @@ function setupPageObserver() {
 
 // cleanup
 window.addEventListener("beforeunload", () => {
+  cleanupPageObserver();
+  cleanupFeedObserver();
+  cleanupWatchObserver();
```

Comment:
Is the order wrong? Since if you call `cleanupPageObserver()` first then the `cleanupFeedObserver()` will no no-op .

Should it be,
```
  cleanupFeedObserver();
  cleanupWatchObserver();
  cleanupPageObserver();
```

Reply:
The observers and the variables holding them are independent so the order shouldn't matter. Do you want to change that?

Reply:
Oh sorry I was a dummy. I saw a check for `if (feedObserver !== null)` in `cleanupPage` but that was deleted. lol